src/helpers/reviews.js

```diff
@@ -1,15 +1,23 @@
 import fetch from "node-fetch";
 
-const API_URL_REVIEWS = "https://plankton-app-xhkom.ondigitalocean.app/api/reviews";
+const API_URL_REVIEWS =
+  "https://plankton-app-xhkom.ondigitalocean.app/api/reviews";
 
-export async function loadMovieReviews(id) {
-  const res = await fetch(`${API_URL_REVIEWS}?filters[movie]=${id}`);
+export async function loadMovieReviews(id, query) {
+  let res = "";
+  if (query.page != undefined) {
+    res = await fetch(
+      `${API_URL_REVIEWS}?filters[movie]=${id}&pagination[page]=${query.page}&pagination[pageSize]=5`
+    );
+  } else {
+    res = await fetch(`${API_URL_REVIEWS}?filters[movie]=${id}`);
+  }
   const payload = await res.json();
-  return payload.data;
+  return payload;
 }
 
 export async function loadAllReviews() {
   const res = await fetch(API_URL_REVIEWS);
   const payload = await res.json();
-  return payload.data;
-}
\ No newline at end of file
+  return payload;
```

Comment:
What is the purpose of this function? Is there a feature that requires it to be possible to load all reviews?

I don't think so, and if I'm right, I would remove this code. It's always better to not have "dead code" than to have it lying around, because it can be confusing and cause programmers to make mistakes in live code.